src/api/procedures/types.ts

```diff
@@ -665,7 +665,7 @@ export interface CreateAssetParams {
    *   {@link types!KnownAssetType} enum, but custom values can be used as well. Custom values must be registered on-chain the first time
    *   they're used, requiring an additional transaction. They aren't tied to a specific Asset
    */
-  assetType: KnownAssetType | string;
+  assetType: KnownAssetType | string | BigNumber;
```

Comment:
Description needs to be modified to specify what BigNumber value can be passed here